src/vivarium_nih_us_cvd/components/effects.py

```diff
@@ -96,21 +96,6 @@ def _polypill_sbp_adherence_modifier(
         return target
 
 
-class PAFCalculationRiskEffect(RiskEffect):
```

Comment:
I just moved this down below MediatedRiskEffect so that it can inherit it (but I otherwise didn't touch it)